examples/dt/compute/amd-sev/edpm/nova_amd.yaml

```diff
@@ -0,0 +1,26 @@
+#
+#test amd sev 
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: amd-sev
+data:
+  01-amd-sev.conf: |
```

Comment:
To be on the safe side I would use a bigger sequence number than `01`
Currently two files are automatically generated for the nova-compute services:
```
[root@edpm-compute-0 ~]# podman exec -it nova_compute ls -alh /etc/nova/nova.conf.d/
total 12K
drwxr-xr-x. 2 root root  109 Jan 22 16:03 .
drwxr-xr-x. 1 root root   42 Jan 22 16:03 ..
-rw-------. 1 nova nova 4.7K Jan 22 16:03 01-nova.conf
-rw-------. 1 nova nova   34 Jan 22 16:03 02-nova-host-specific.conf
```
* 01-nova.conf is the cell specific infra configuration and all of our default configuration from the nova-operator
* 02-nova-host-specific.conf is not cell but host specific. Today it o nly contains the IP of the host.

The oslo.config loads the config files in lexicographical order. So if you want to make sure that your config overrides our defaults then I suggest to use filename with bigger than 02 prefix.

Reply:
ack

Reply:
we want to resserve config below 25 for future internal use

so this should be renamed to 

25-amd-sev-nova.conf

the code we use to copy the config files is also checking for files with *nova*.conf so the current form wont be picked up